includes/html/pages/device/services.inc.php

```diff
@@ -67,33 +67,48 @@
     // Loop over each service, pulling out the details.
 
     echo '<table class="table table-hover table-condensed">';
+    echo '<thead>';
+    echo '<th style="width:1%;max-width:1%;"></th>';
```

Comment:
Hardcoded width percents is not the best idea.  It looks good in your screenshot, but will not look good a different screen resolutions.

Reply:
I just copied this over from the All Services page.

Reply:
Pretty much, yes!
![IMG_5AAAC7CC7BF9-1](https://github.com/user-attachments/assets/97f38e51-e735-4ec8-89c4-5a5a2a059cb2)